src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3790,6 +3790,71 @@ internal sealed partial class LazyCoalesceExpression : BaseCoalesceExpression, I
         public override IOperation WhenNull => SetParentOperation(_lazyWhenNull.Value, this);
     }
 
+    internal abstract partial class BaseCoaleseAssignmentOperation : Operation, ICoalesceAssignmentOperation
+    {
+        protected BaseCoaleseAssignmentOperation(bool isChecked, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(OperationKind.CoalesceAssignment, semanticModel, syntax, type, constantValue, isImplicit)
+        {
+            IsChecked = isChecked;
+        }
+
+        public abstract IOperation Target { get; }
+        public abstract IOperation WhenNull { get; }
+        public bool IsChecked { get; }
+
+        public override IEnumerable<IOperation> Children
+        {
+            get
+            {
+                if (Target != null)
```

Comment:
>Target [](start = 20, length = 6)

Can ```Target``` ever be null?

Reply:
Doesn't look like it. I'll add asserts to the verifier, but I'm going to keep the `if` checks as we do that with every other field in this file.

---
In reply to: [209382051](https://github.com/dotnet/roslyn/pull/29063#discussion_r209382051) [](ancestors = 209382051)

Reply:
I'll also add a test for otherwise empty left and right nodes.

---
In reply to: [210781658](https://github.com/dotnet/roslyn/pull/29063#discussion_r210781658) [](ancestors = 210781658,209382051)